### PR TITLE
setup Release Notes and add content for DOC-4415

### PR DESCRIPTION
**Pull request by nadinequinr3**
https://r3-cev.atlassian.net/browse/DOC-4415

### CODE REVIEW

**Review by paulmoloneyr3 — APPROVED** [review]

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-release-notes
+    weight: 6000
+section_menu: corda-5-beta
+---
+
+Corda 5.0 Beta is a pre-release version for testing purposes only.
+{{< note >}}
+If you are not part of the beta programme, the Corda 5.0 Beta documentation is for information only.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md:13]:
```suggestion
If you are not part of the Beta program, the Corda 5.0 Beta documentation is for information only.
```

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by driessamyn — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  corda-5-beta:
+    identifier: corda-5-beta-release-notes
+    weight: 6000
+section_menu: corda-5-beta
+---
+
+Corda 5.0 Beta is a pre-release version for testing purposes only.
+{{< note >}}
+If you are not part of the Beta program, the Corda 5.0 Beta documentation is for information only.
+R3 will be running a Beta Program for Corda 5.0 beginning in 2023. Contact R3 to register your interest.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md:14]:
```suggestion
R3 will be running a Beta program for Corda 5.0 beginning in 2023. Contact R3 to register your interest.
```
Is it Beta Program or Beta program? (See the sentence above.)

**Review by paulmoloneyr3 — APPROVED** [review]

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md (hunk 2 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md:14]: My take was if we're capitalising "beta" we're saying the "Beta Program" like it's the name of something. If we don't capitalise beta it's just describing the program.
image
Open to correction though.. @AnnaK-R3 @paulmoloneyr3 @colin-hillis

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-release-notes
+    weight: 6000
+section_menu: corda-5-beta
+---
+
+Corda 5.0 Beta is a pre-release version for testing purposes only.
+{{< note >}}
+If you are not part of the Beta program, the Corda 5.0 Beta documentation is for information only.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md:13]:
```suggestion
If you are not part of the Beta Program, the Corda 5.0 Beta documentation is for information only.
```

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0-beta/release-notes/release-notes-c5.md:14]: Ok, I understand. In this case, please close this comment and apply the change for the sentence above.

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**